pkg/epforwarder/epforwarder_mockgen.go

```diff
@@ -0,0 +1,101 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
I generated it directly from IDE:
![image](https://user-images.githubusercontent.com/49917914/228491338-562faf1b-b8fa-4211-8e13-9ef6fcc65a38.png)

that runs this command (can be run manually):
```
go generate pkg/epforwarder/epforwarder.go
```

Do you think I should make it an invoke task like this using for example `invoke epforwarder.gen-mocks`?
https://github.com/DataDog/datadog-agent/blob/fa68a1a14b6642fabe4916a140b4846d1f146ce1/tasks/process_agent.py#L153-L170

Reply:
IMO the best would be for it to be part of an invoke task yes. If the generation is idempotent (I hope?), it might even be better to have this said task executed when running `agent build`, WDYT?

Reply:
Just added it here as `inv epforwarder.gen-mocks`:

https://github.com/DataDog/datadog-agent/pull/16174/commits/4c408b25f06db412c1697b29551e00339c5032ba


> If the generation is idempotent (I hope?), it might even be better to have this said task executed when running agent build, WDYT?

My concern with is that the output can change if new versions of mockgen library change the way it generate the final output this will trigger code change and might be noisy for other teams.

I think it might be better to keep it manual since we usually only need to regenerate it if the interface EventPlatformForwarder changes.

Reply:
> My concern with is that the output can change if new versions of mockgen library change the way it generate the final output this will trigger code change and might be noisy for other teams.

Depends on what you want: being kept up-to-date with the mock library changes or having to deal with upgrades once in a while. Your call, I've no strong opinion in this case.